Flora-Fauna-Frontend/src/components/LandingPage/LandingPage.jsx

```diff
@@ -15,7 +15,6 @@ export default function LandingPage() {
    //Sends input value to be requested, then navigates to search page, which will display the results
    function handleOnSubmit(){
       //Send search request
-      console.log("hello")
```

Comment:
Comment out

Reply:
RECOMMENDED: Use Javascript Console Utils for making and cleaning up your console log statements.